packages/salesforcedx-vscode-core/src/messages/i18n.ja.ts

```diff
@@ -326,7 +326,7 @@ export const messages = {
   CompactLayout: 'Compact Layouts',
   ConnectedApp: 'Connected Apps',
   ContentAsset: 'Content Assets',
-  CorsWhitelistOrigin: 'Cors Whitelist Origins',
+  CorsAllowlistOrigin: 'Cors Allowlist Origins',
```

Comment:
I think this is an metadata `xmlName`. Will this need to be changed in sdr lib too? https://github.com/forcedotcom/source-deploy-retrieve/blob/6eedb130caaec33de64111b99f4d7edcf880265e/src/metadata-registry/data/registry.json#L1174 @brpowell

Reply:
@xyc @LisandroFernandezSF Agreed, I think this should remain as it was.

Reply:
Yup, should stay the same but still good find. On the documentation page for this type they say allowlist, but the type name itself stays for API reasons.

Reply:
I understand the property  `CorsWhitelistOrigin` should stay, but if I change only the value, like:  `CorsWhitelistOrigin: 'Cors Allowlist Origins'`, is that OK?